### DISCUSSION TITLE
reject completions?

### DISCUSSION THREAD

**Discussion by nbecker** [minad/vertico#457] [Q&A] [answered]
Perhaps not an issue, just a question from a newbie.
I'm in dired looking at:
```
  lrwxrwxrwx. 1 nbecker nbecker    15 Nov 28  2022 pybind11 -> pybind11-2.10.1
  drwxrwxr-x. 1 nbecker nbecker   334 Oct 31  2022 pybind11-2.10.1
  drwxrwxr-x. 1 nbecker nbecker   318 Apr 30  2018 pybind11-2.2.3
  drwxrwxr-x. 1 nbecker nbecker   318 Jun 11  2019 pybind11-2.3.0
  drwxrwxr-x. 1 nbecker nbecker   318 Jan  4  2021 pybind11-2.5.0
```

I want to make that symlink pybind11 using dired-do-relsymlink (assume it isn't there already).  If I type 'pybind11', I will have a default completion on pybind11-2.10.1, which I don't want.  I don't want any completion, I just want 'pybind11' which doesn't yet exist.  If I hit RET it will take the default completion pybind11-2.10.1 which is wrong.  How can I reject the completion and just accept my typed input?

My vertico setup is pretty basic
```
;; Enable vertico
(use-package vertico
  :init
  (vertico-mode)
  :custom
  (vertico-sort-function 'vertico-sort-history-alpha))
;; Enable richer annotations using the Marginalia package
  (use-package marginalia
    :bind (:map minibuffer-local-map
	   ("M-A". marginalia-cycle))
    :init
    (marginalia-mode))
```

**Comment by minad** [comment, marked as answer]
The input can be submitted with M-RET. See https://github.com/minad/vertico?tab=readme-ov-file#key-bindings.
